events-api/src/modules/events/events.controller.ts

```diff
@@ -47,4 +46,19 @@ export class EventsController {
       data: foundEvent,
     };
   }
+
+  @Patch(':eventId')
```

Comment:
Can everyone update an event?

Reply:
fixed: authenticated users with the role of `admin` can update